### ISSUE TITLE
Reverb roomsize issue and level/width dependency

### ISSUE THREAD

**Issue by jjceresa** [FluidSynth/fluidsynth#279]
Hello,
1) I have noticed one minor issue on upper limit `roomsize` setting for value above 1.
Actually the command handler propose an `upper limit` value `1.2`.
For value `above 1.07`, the output amplitude grow exponentially. This entering the user in high stress :).
An upper limit of `1.0 `seems sufficient as it produce a long `reverb time` (an can even ovoid  load speakers damages).
***
2) `Width `is a nice setting for value above 0 (`[0..3]`)  as we benefit of the `internal stereo effect `of the unit.
However for value in the upper range `[1..100]`, the` output amplitude` augment (rather linearly versus `width`).
Actually for `width value > 50`, the output amplitude can be clipped by the audio card . This make also unhappy musician.
Of course the user can compensate by changing the `level `setting to lower value .This is not easy as he must try multiples times before finding the right level.

We can also compensate automatically the output amplitude (i.e `rev->wet`) like this:

```
//#define scalelevelwidth 0.16
#define scalelevelwidth 0.2
		rev->wet = (rev->level ) /   (1 + rev->width * scalelevelwidth);
```

Actually, with a  `scalelevelwidth `of 0.2, for a given level setting (i.e `rev->level` ), the output amplitude `rev->wet` seems rather independent of  `width` setting (i.e`rev->width`).
***
I can easily  do a Pull Request if you want.

**Comment by derselbst** [comment]
Regarding 1. I'm absolutely with you, I also ran into those kind of problems. Regarding 2. we'll have to see if this has a too big impact on how the reverb response to user settings. So yeah a PR would be nice.

**Comment by jjceresa** [comment]
> Regarding 2. we'll have to see if this has a too big impact on how the reverb response to user settings. 

I was waiting for this question. 
Applying this compensation doesn't impact the `width `effect response but only the `scale `of the  width. 

- In others words for some user already `existing configurations settings` (particularly those with `width > 3`), it is possible that the user will need to augment the `Width `value and re-augment the `level `settings

- For future `configurations width` (those width > 3), it should be easier to build a `new width configuration` from a `previous width configuration`. Because the user should not be obliged to compensate  drastically and empirically on the level setting.

- For others old configurations (with width < 3), i had heard a negligible impact.

Anyway i will create a branch `reverb-settings-adjust` and 2 commits

- One commit with the roomsize limit.
- The next commit will include the `level-width compensation`.

This will not to difficult to verify playing a single note of percussive instrument (at full velocity) (i.e piano preset 1 or 2 of any GM SoundFont).

**Comment by derselbst** [comment]
Fixed by merging #280

**Comment by jjceresa** [comment]
Ok, thanks.
